CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,184 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Should this say `Nautobot Community Code of Conduct` instead?

Reply:
This is a community standard thing. https://www.contributor-covenant.org/version/2/1/code_of_conduct/